node/src/Transaction.ts

```diff
@@ -2389,6 +2390,30 @@ export class Transaction extends BaseTransaction<Transaction> {
     public select(index: number): Transaction {
         return this.addAndReturn(createSelect(index));
     }
+
+    /**
+     * Copies the value stored at the `source` to the `destination` key on `destinationDB`. When `replace`
```

Comment:
Can we use the same docs as for GlideClient here?
```suggestion
     * Copies the value stored at the `source` to the `destination` key on `destinationDB`. If `destinationDB` is specified,
     * the value will be copied to the databased specified, otherwise the current database will be used. When `replace`
```